server/integration_tests/test_data/international/query_3/chart_config.json

```diff
@@ -104,34 +104,6 @@
             ],
             "title": "Unemployment Rate"
           },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "UnemploymentRate_Person_Female"
-                    ],
-                    "title": "Unemployment Rate of Females in United Kingdom",
-                    "type": "LINE"
-                  }
-                ]
-              },
-              {
-                "tiles": [
-                  {
-                    "description": "Unemployment Rate of Females in United Kingdom",
-                    "statVarKey": [
-                      "UnemploymentRate_Person_Female"
-                    ],
-                    "title": "Unemployment Rate of Females in United Kingdom",
-                    "type": "HIGHLIGHT"
-                  }
-                ]
-              }
-            ],
-            "title": "Unemployment Rate of Females"
```

Comment:
Slightly sad.. Is this from the non-worker going on top?

Reply:
I am actually unsure why this has happened because no new chart seems to have come up. Only explanation I have for this is that the dc/topic/Income (to which a two new SVs got added) is part of dc/topic/Economy which does match in the detection. But otherwise, no other changes...